toonz/sources/toonz/xsheetviewer.cpp

```diff
@@ -412,18 +404,11 @@ void XsheetViewer::positionSections() {
                         bodyLayer.adjusted(0, -XsheetGUI::SCROLLBAR_WIDTH)));
   m_rowScrollArea->setGeometry(o->frameLayerRect(
       bodyFrame.adjusted(0, -XsheetGUI::SCROLLBAR_WIDTH), headerLayer));
-  if (o->isVerticalTimeline())
-    m_frameZoomSlider->hide();
-  else {
-    if (m_frameZoomSlider->isHidden()) m_frameZoomSlider->show();
-    QRect sliderRect =
-        QRect(0, 0, XsheetGUI::ZOOM_SLIDER_WIDTH,
-              XsheetGUI::SCROLLBAR_WIDTH - 2)
-            .translated(m_columnScrollArea->rect().right() -
-                            XsheetGUI::ZOOM_SLIDER_WIDTH - 17,
-                        height() - XsheetGUI::SCROLLBAR_WIDTH + 1);
-    m_frameZoomSlider->setGeometry(sliderRect);
-  }
+
+  m_layerFooterPanel->setGeometry(0, m_columnScrollArea->height() + 50,
+                                  m_columnScrollArea->width(), 14);
```

Comment:
For now the slider seems to offset upward to the incorrect position when the xsheet toolbar is visible.

How about changing this line like as follows?:
```
  m_layerFooterPanel->setGeometry(0, m_columnScrollArea->geometry().bottom()+1,
                                  m_columnScrollArea->width(), 14);
```